### ISSUE TITLE
Добавить счётчик символов для поста-ссылки

### ISSUE THREAD

**Issue by Finitumus** [ImaginariaRU/dear-imaginaria#29]
Длина поста-ссылки сейчас ограничена 500 символами, и пока не попытаешься его отправить, не видно, влезает в них пост или нет.
Хотелось бы, чтобы, как у записи на стене, под полем ввода поста был счётчик, показывающий, сколько символов осталось до 500.

**Comment by KarelWintersky** [comment]
Так есть же. Только не интерактивный.

**Comment by Finitumus** [comment]
Так, как есть сейчас, неудобно, иначе я не стала бы поднимать тут issue. 
Я прошу сделать интерактивный, как на стене.

**Comment by KarelWintersky** [comment]
Принято. Будем думать.
